.github/workflows/ci.yml

```diff
@@ -34,20 +41,20 @@ jobs:
           python -m pip list
       - name: Run python lint
         run: pylint lektor
-      - name: Cache pip
-        uses: actions/cache@v1
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
 
   lint-js:
     name: Lint JS
     runs-on: ubuntu-latest
     steps:
       - name: Checkout branch
         uses: actions/checkout@v2
+      - name: Cache node
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
```

Comment:
I think you can even go as far as saving node_modules itself.

Reply:
Do I need to add the node version to the cache key as well @runfalk ?

Reply:
I'm not sure but I think it's safest to do so.

Reply:
Fixed